src/apis/nimbus.js

```diff
@@ -126,11 +125,7 @@ var nimbus = class extends ExtensionAPI {
 
           async evaluateJEXL(expression, context = {}) {
             try {
-              const targetingContext = await new TargetingContext(context, {});
-
-              const result = await targetingContext.evalWithDefault(expression);
-
-              return result;
+              return await lazy.FilterExpressions.eval(expression, context);
```

Comment:
Why this change?

Reply:
`targetingContext.evalWithDefault` seems to use the actual values of the user's os, locale, and a couple of other items rather than the values from the changed context that gets passed into TargetingContext and I found that `lazy.FilterExpressions.eval` only uses the values from the context passed into it so I made that change